### DISCUSSION TITLE
It would be nice to specify, as a command-line parameter, the number of lines to import from a file...

### DISCUSSION THREAD

**Discussion by jwnacnud** [saulpw/visidata#2355] [Ideas]
If I have a large file, certainly I can just HEAD it to get the lines I want.  I'd love to run the following:

vd -n1000 filename.csv

and have visidata simply open the first 1000 records.  I know I can ctrl-c to stop along the way on a large file.  

Am I just dreaming?

**Comment by saulpw** [comment]
Hi @jwnacnud, that feature doesn't exist yet.  You could open an issue that would become a wishlist item, and it might be implemented at some point.  I'm curious though, why would you only want the first 1000 records?  Why not 10k or 100k?

**Comment by jwnacnud** [comment]
Thanks.  Done.

Jeffrey Duncan
